tests/trestle/tasks/oscal_profile_to_osco_profile_test.py

```diff
@@ -0,0 +1,283 @@
+    """Test print_info call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile.config')
+    config.read(config_path)
+    section = config['task.oscal-profile-to-osco-profile']
+    section['output-dir'] = str(tmp_path)
+    tgt = oscal_profile_to_osco_profile.ProfileToOsco(section)
+    retval = tgt.print_info()
+    assert retval is None
+
+
+def test_oscal_profile_to_osco_profile_simulate(tmp_path):
+    """Test simulate call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile.config')
+    config.read(config_path)
+    section = config['task.oscal-profile-to-osco-profile']
+    section['output-dir'] = str(tmp_path)
+    tgt = oscal_profile_to_osco_profile.ProfileToOsco(section)
+    retval = tgt.simulate()
+    assert retval == TaskOutcome.SIM_SUCCESS
+    assert len(os.listdir(str(tmp_path))) == 0
+
+
+def test_oscal_profile_to_osco_profile_execute(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path('tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile.config')
+    config.read(config_path)
+    _test_oscal_profile_to_osco_profile_execute_common(tmp_path, config)
+
+
+def test_oscal_profile_to_osco_profile_execute_osco_0_1_39_parms_no(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(
+        'tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile-0.1.39-parms-no.config'
+    )
+    config.read(config_path)
+    _test_oscal_profile_to_osco_profile_execute_common(tmp_path, config)
+
+
+def test_oscal_profile_to_osco_profile_execute_osco_0_1_39_parms_yes(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(
+        'tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile-0.1.39-parms-yes.config'
+    )
+    config.read(config_path)
+    _test_oscal_profile_to_osco_profile_execute_common(tmp_path, config)
+
+
+def test_oscal_profile_to_osco_profile_execute_osco_0_1_40_parms_no(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(
+        'tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile-0.1.40-parms-no.config'
+    )
+    config.read(config_path)
+    _test_oscal_profile_to_osco_profile_execute_common(tmp_path, config)
+
+
+def test_oscal_profile_to_osco_profile_execute_osco_0_1_40_parms_yes(tmp_path):
+    """Test execute call."""
+    config = configparser.ConfigParser()
+    config_path = pathlib.Path(
+        'tests/data/tasks/oscal-profile-to-osco-profile/oscal-profile-to-osco-profile-0.1.40-parms-yes.config'
+    )
+    config.read(config_path)
```

Comment:
Those three lines repeat in each test with a different string for config file. Please create a utility function to create and read config

Reply:
Or it is a good case for parametrizing the test.  See e.g. folders_test.py